Week-1/index.html

```diff
@@ -0,0 +1,102 @@
+<dom>
+    <!DOCTYPE html>
+    <html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet"
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+
+    <body>
+        <aside>
+            <div class="sidebar">
+                <div><i class="fa fa-bars"></i></div>
+                <div><i class="fa fa-home"></i></div>
+                <div><i class="fa fa-search"></i></div>
+                <div><i class="fa fa-volume-up"></i></div>
+                <div><i class="fa fa-user"></i></div>
+                <div><i class="fa fa-cog"></i></div>
+                <div><i class="fa fa-spotify"></i></div>
+                <div><i class="fa fa-soundcloud"></i></div>
+                <div><i></i></div>
+            </div>
+        </aside>
+
+        <div class="header">
+            <div class="div-left-header">
+                <img
+                    src="https://uploads.codesandbox.io/uploads/user/a8233562-b36b-483d-9de9-5a1ac148fb0c/2Eq--cattyboard.jpg">
+            </div>
+            <div class="div-right-header">
+                <div class="div-right-header-top">
+                    <h3>CattyBoard Top 100 Single Charts(11.06.36) </h3>
+                    <h5>Unknown Artist <br> 2016.Charts.100 songs</h5>
+                </div>
+                <div class="div-right-header-bottom">
+                    <i class="fa fa-play width"> Play all </i><i class="fa fa-plus width"> Add to</i> <i
+                        class="fa fa-ellipsis-h"> Move</i>
+                </div>
+            </div>
+        </div>
+
+        <div class="div-body">
+            <ul id="menu">
```

Comment:
@soumya213 can this be achieved by one single ui and four li elements ?

Something like this "<ul><li><p>....</p></li></ul>"